pages/index.tsx

```diff
@@ -10,32 +11,50 @@ import { AUTH_COOKIE_NAME } from 'shared/constants';
 
 interface Props {
   isAuthed: boolean;
+  seenOnBoarding: boolean;
 }
 
 const Index: NextPage<Props> = props => {
   const [isAuthed, setIsAuthed] = React.useState(props.isAuthed);
+  const [seenOnBoarding, setSeenOnBoarding] = React.useState(props.seenOnBoarding);
+  const router = useRouter();
+
+  // to avoid this error:
+  // During Pre-rendering (SSR or SSG) you tried to access a router method push, replace, back, which is not supported.
+  React.useEffect(() => {
+    const { query } = router;
+    if (query.seenOnBoarding === 'true') {
+      setSeenOnBoarding(true);
+      // reset query parameters once user gets back to home page
+      router.replace({
+        pathname: '/',
+        query: {},
+      });
+    } else if (!seenOnBoarding && isAuthed) {
+      router.push('/onBoarding');
+    }
+  }, []);
```

Comment:
Highly influenced by ChatGPT 👀

Let me know if there is a better way, Next is confusing

Reply:
i'll check it out today! thanks!

Reply:
Be careful with scope creep. If I'm understanding #76 correctly, this ticket should redirect to onboarding after each login, unconditionally.

Keeping track of if the user has seen the onboarding page is not a part of this ticket. We need #78 to create the data structure to track that and #79 will come back to implement it.

_As a side note, this logic should probably have been in `client/pages/auth-redirect` since it determines where we go after a login and doesn't need executed everytime we hit the homepage._

Reply:
My bad, I didn't realize this was scope creep at all, I thought I was doing the bare minimum to get the feature working. 

I'll make the changes, I didn't understand how the auth-redirect route was getting hit but ill look into it more.